recipes/pulsar_phase/pulsar_phase_computation.ipynb

```diff
@@ -0,0 +1,1895 @@
+     ]
+    }
+   ],
+   "source": [
+    "phase_log = get_log(ephemeris_file=ephemeris_file, phase_column_name = 'PHASE')\n",
+    "print(phase_log)"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 26,
+   "id": "5a3e56d2",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# Add the generated string to the meta data of the table\n",
+    "table.meta['PH_LOG'] = phase_log"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "26592cbb",
+   "metadata": {},
+   "source": [
+    "Once this is done, we can put back the table in a new `EventList` object and in a new `Observation` object. "
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 29,
+   "id": "866007b0",
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "EventList\n",
+      "---------\n",
+      "\n",
+      "  Instrument       : MAGIC stereo\n",
+      "  Telescope        : MAGIC\n",
+      "  Obs. ID          : 5029747\n",
+      "\n",
+      "  Number of events : 11189\n",
+      "  Event rate       : 9.417 1 / s\n",
+      "\n",
+      "  Time start       : 56569.18112772242\n",
+      "  Time stop        : 56569.19487901596\n",
+      "\n",
+      "  Min. energy      : 1.06e-02 TeV\n",
+      "  Max. energy      : 4.71e+01 TeV\n",
+      "  Median energy    : 7.92e-02 TeV\n",
+      "\n",
+      "  Max. offset      : 2.6 deg\n",
+      "\n"
+     ]
+    }
+   ],
+   "source": [
+    "# create new event list and add it to observation object\n",
+    "new_event_list = EventList(table)\n",
+    "observation._events = new_event_list\n",
+    "print(observation.events)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "934d147a",
+   "metadata": {},
+   "source": [
+    "## 4. Save new Event List and writing a modify HDU index table"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "e369d2dc",
+   "metadata": {},
+   "source": [
+    "In the following, we show how to write the files in a directory contained in the original datastore directory. This follows the logic of DL3 data store and facilitate the manipulation of the HDU table.\n",
+    "\n",
+    "If one does not want to save the events files and directly perform the pulsar analysis, this step is not required as well as the step of the meta data handeling. However, be aware that for large dataset, the computation of phases can take tens of minutes. "
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 30,
+   "id": "f3c8410c",
+   "metadata": {},
+   "outputs": [
+    {
+     "data": {
+      "text/plain": [
+       "PosixPath('/Users/alvarom/Desktop/Doctorado/fermipulsar/gammapy-datasets/1.0/magic/rad_max/data')"
```

Comment:
printing this is not very useful. this can be skipped

Reply:
We show that because there you can see that the EventList does not need to be in the same path as the hdu_table. I think it could help to follow the explanation.